auto_tests/network_test.c

```diff
@@ -28,7 +28,7 @@ START_TEST(test_addr_resolv_localhost)
     int localhost_split = 0;
 
     IP ip;
-    ip_init(&ip, 0);
+    ip_init(&ip, /* ipv6? */ 0);
```

Comment:
can you append this to the end of the line, and not inside the function? (and elsewhere too)